mmcv/fileio/file_client.py

```diff
@@ -191,6 +191,20 @@ def get_text(self, filepath):
         return value_buf
 
 
+class HTTPBackend(BaseStorageBackend):
+    """HTTP and HTTPS storage bachend."""
+
+    def get(self, filepath):
+        from urllib.request import urlopen
```

Comment:
Since urllib is a builtin package, we may just put it at the front of file.

Reply:
Got it.